staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1/types.go

```diff
@@ -104,6 +104,8 @@ type APIServiceCondition struct {
 type APIServiceStatus struct {
 	// Current service state of apiService.
 	// +optional
+	// +patchMergeKey=type
```

Comment:
What is the problem of this one? It copied from the struct tag directly, so if it is not right, then struct tag is not right too.